member/src/main/resources/sample-requests.txt

```diff
@@ -5,7 +5,7 @@
  grpcurl -plaintext localhost:8081 list
 
 //RegisterMember
-grpcurl -plaintext -d '{"member_id": {"id": "memberId"}, "memberInfo":{"handle":"test", "avatarUrl": "N/A", "firstName":"Shail", "lastName": "Verma", "contact" : {"firstName": "firstName", "lastName": "lastName", "phone": "111-222-1234", "emailAddress": "test@example.com", "userName":"ShailUser"}, "notificationPreference": "NOTIFICATION_PREFERENCE_EMAIL", "notificationOptIn": true, "organizationMembership": [{"id":"ORGANIZATION_IMPROVING"}], "tenant": {"id":"TENANT_IMPROVING"}}, "registeringMember":{"id":"ADMIN"} }' localhost:8081 com.improving.app.member.api.MemberService.RegisterMember
+grpcurl -plaintext -d '{"member_id": {"id": "memberId"}, "memberInfo":{"handle":"test", "avatarUrl": "N/A", "firstName":"Shail", "lastName": "Verma", "contact" : {"firstName": "firstName", "lastName": "lastName", "phone": "111-222-1234", "emailAddress": "test@example.com", "userName":"ShailUser"}, "notificationPreference": "NOTIFICATION_PREFERENCE_EMAIL", "notificationOptIn": true, "organization_membership": [{"id":"ORGANIZATION_IMPROVING"}], "tenant": {"id":"TENANT_IMPROVING"}}, "registeringMember":{"id":"ADMIN"} }' localhost:8081 com.improving.app.member.api.MemberService.RegisterMember
```

Comment:
I didn't notice this change. I was changing snake case to camel case because camel case is preferred as per the protobuf spec, and automatically converts to camel case for Scala derivations. This change has not introduced a failure, but I will make sure wherever it's used it's actually being tested.